client/a8c-for-agencies/components/guided-tour-step/index.tsx

```diff
@@ -71,13 +71,13 @@ export function GuidedTourStep( { id, tourId, context, hideSteps, className }: P
 	// Add a click event listener to the context element if the step requires it
 	useEffect( () => {
 		if ( currentStep && currentStep.nextStepOnTargetClick && context ) {
-			context.addEventListener( 'click', nextStep );
+			( context as HTMLElement ).addEventListener( 'click', nextStep );
```

Comment:
I don't think it's necessary here. Should be ok just to use `context.addEventListener`?